src/igvfd/searches/configs/prediction.py

```diff
@@ -0,0 +1,47 @@
+from snovault.elasticsearch.searches.configs import search_config
+
+
+@search_config(
+    name='Prediction'
+)
+def cell_line():
```

Comment:
not a cell line

Reply:
Same copy paste error as in auxiliary set, sorry about this. Fixed.